src/Platforms/SqlitePlatform.php

```diff
@@ -765,7 +765,7 @@ protected function getPostAlterTableIndexForeignKeySQL(TableDiff $diff)
     protected function doModifyLimitQuery($query, $limit, $offset)
     {
         if ($limit === null && $offset > 0) {
-            return $query . ' LIMIT -1 OFFSET ' . $offset;
+            return $query . sprintf(' LIMIT -1 OFFSET %d', $offset);
```

Comment:
Should `$query` also be moved under `sprintf()`? This way, we'll have one variable / memory allocation fewer .

Reply:
Since this is in a `return` line it shouldn't really matter anymore.

Reply:
It still does. In order to return a value, the interpreter needs to allocate a new zval, copy the value of `$query` there, then append the value returned by `sprintf()`, then return it. If it's just `sprintf()`, this could be avoided.

We're of course not trying to save CPU ticks here but a combination of `sprintf()` and `.` on the same line looks dirty to me.

Reply:
Done.

What about the `$query .= sprintf` in all the other platforms, they are okay?